packages/manager/modules/telecom-universe-components/src/telecom/pack/migration/pack-migration-process.service.js

```diff
@@ -216,8 +223,29 @@ export default /* @ngInject */ function($q, OvhApiPackXdsl, Poller) {
       });
   }
 
+  /**
+   * Retrieve contact owner associated to the pack if it exists
+   */
+  function getCurrentContactOwner() {
+    migrationProcess.contactOwner = null;
+    return OvhApiPackXdsl.v6()
+      .getContactOwner({
+        packName: migrationProcess.pack.packName,
+      })
+      .$promise.then((result) => {
+        migrationProcess.contactOwner = result.data;
+        if (migrationProcess.contactOwner) {
+          migrationProcess.contactPhone = migrationProcess.contactOwner.phone;
+        }
+      });
```

Comment:
I think it is now possible to use : 
```suggestion
          migrationProcess.contactPhone = migrationProcess.contactOwner?.phone;
```